installation/get-started-minikube.md

```diff
@@ -1,71 +1,74 @@
 # Minikube
 
-To create a LangStream cluster, you will need [kubectl](https://kubernetes.io/docs/reference/kubectl/), [helm cli](https://helm.sh/docs/intro/install/), and a running K8s cluster (a default 4 CPU [minikube](https://minikube.sigs.k8s.io/docs/start/) is fine).
+To create a LangStream cluster locally, it's recommended to use [minikube](https://minikube.sigs.k8s.io/docs/start/) - setting 4 CPUs is highly recommended.
+`mini-langstream` comes in help for installing and managing your local cluster.
 
-### Installation
 
-1. Start a minikube:
+## Install mini-langstream
 
-```bash
-minikube start --cpus 4
-```
+MacOS:
 
-2. Install MinIO for local testing:
-
-```
-kubectl apply -f https://raw.githubusercontent.com/LangStream/langstream/main/helm/examples/minio-dev.yaml
+```bash
+brew install LangStream/langstream/mini-langstream
 ```
 
-3. Deploy Langstream from the Helm repository:
+Unix:
 
 ```bash
-helm repo add langstream https://langstream.github.io/charts
-helm repo update
-helm upgrade \
-    -i langstream \
-    -n langstream \
-    --create-namespace \
-    --values https://raw.githubusercontent.com/LangStream/langstream/main/helm/examples/simple.yaml \
-    langstream/langstream
+curl -Ls "https://raw.githubusercontent.com/LangStream/langstream/main/mini-langstream/get-mini-langstream.sh" | bash
 ```
 
-4\. Open the control-plane and api-gateway ports:
+mini-langstream requires the following commands to be already installed:
+- Docker
+- Minikube
+- Helm
+- Kubectl
+- LangStream CLI
 
-```bash
-kubectl -n langstream port-forward svc/langstream-control-plane 8090:8090 &
-kubectl -n langstream port-forward svc/langstream-api-gateway 8091:8091 &
-```
 
 {% hint style="info" %}
-The ampersand (&) keeps the port forward running as a background process.
+If you install mini-langstream using Brew, all the dependencies are automatically installed.
 {% endhint %}
 
-### Environment setup
 
-1. Port forward minio in a separate terminal:
+## Start the cluster
 
-```
-kubectl port-forward pod/minio 9000:9090 -n minio-dev  
+mini-langstream will do all the setup for you, in particular:
+- it starts `minikube` in a dedicated context
+- it deploys LangStream components using `helm`
+- it runs a stateful Kafka broker as docker container
+- it runs a stateful s3-compatible storage (MinIO) as docker container
```

Comment:
we should add the database here

Reply:
it doesn't start herddb at the moment, we can add it